Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -920,7 +920,9 @@ eval (SearchLocationModelActionController (SearchLocationModelController.UpdateS
 eval (QuoteListModelActionController (QuoteListModelController.QuoteListItemActionController (QuoteListItemController.Click quote))) state = do
   continueWithCmd (state { data { quoteListModelState = map (\x -> x { selectedQuote = (Just quote.id) }) state.data.quoteListModelState }, props { selectedQuote = Just quote.id } })
     [ do
-        if (getValueToLocalStore AUTO_SELECTING) == "CANCELLED_AUTO_ASSIGN" then
+        _ <- pure $ setValueToLocalStore AUTO_SELECTING_DURATION $ if (quote.id /= fromMaybe "" state.props.selectedQuote) then quote.timer else getValueToLocalStore AUTO_SELECTING_DURATION
+        autoSelectFlag <- getValueToLocalStoreEff AUTO_SELECTING
```

Comment:
I think `getValueToLocalStoreEff` this is not working fine change it to `getValueToLocalStore`

Reply:
done